docs/examples/config-provider/message.vue

```diff
@@ -10,11 +10,15 @@
 import { reactive } from 'vue'
 import { ElMessage } from 'element-plus'
 
+import type { MessagePlacement } from 'element-plus'
+
 const config = reactive({
   max: 3,
   plain: true,
+  placement: 'bottom' as MessagePlacement,
```

Comment:
Now the casting can be removed.

```suggestion
  placement: 'bottom',
```